package.json

```diff
@@ -76,15 +76,15 @@
     "karma-jasmine": "~2.0.1",
     "karma-jasmine-html-reporter": "^1.4.2",
     "mdi": "^2.2.43",
-    "ng-packagr": "^5.5.1",
+    "ng-packagr": "^9.0.0",
     "npm-run-all": "^4.1.5",
     "protractor": "~5.4.2",
     "release-it": "^12.4.2",
-    "rxjs": "~6.5.3",
-    "ts-node": "~8.4.1",
+    "rxjs": "~6.5.4",
+    "ts-node": "~8.5.4",
     "tslib": "^1.10.0",
     "tslint": "~5.20.0",
-    "typescript": "~3.5.3",
+    "typescript": "~3.7.5",
```

Comment:
Angular 9 is not yet compatible with typescript 3.7.5 please use ng update in the future when trying to make changes like this.